### PR TITLE
 fix: displayName is not set for the component of "@yamada-ui/pagination" #2889 

### PR DESCRIPTION
**Pull request by Utkarshrajmishra**
Closes https://github.com/yamada-ui/yamada-ui/issues/2889
### Description

Set displayName for the @yamada-ui/pagination component to make debugging easier.
This change improves the ability to identify the component name in development tools.

### Current behavior (updates)

Currently, displayName is not set for the @yamada-ui/pagination component, making it difficult to identify the component in debugging tools like React Developer Tools.
New behavior

With displayName set, the @yamada-ui/pagination component now displays its name correctly in debugging tools like React Developer Tools, making it easier for developers to identify and debug the component.

### Is this a breaking change (Yes/No):
No.
### Additional Information
None.

### CODE REVIEW

**Review by illionillion — CHANGES_REQUESTED** [review]
@Utkarshrajmishra 
Thank for PR!!

Please Modify `pagination-item.tsx` and `pagination-icon.tsx` in the same way.
Import `FC` from `@yamada-ui/core` for normal components that do not use `forwardRef`

**Comment by Utkarshrajmishra** [comment]
Hey @illionillion, I have made the required changes. Please have a look.

File: packages/components/pagination/src/pagination-icon.tsx
Comment (illionillion) [inline packages/components/pagination/src/pagination-icon.tsx:1]: @Utkarshrajmishra 
Please import `FC` from `@yamada-ui/core`

File: packages/components/pagination/src/pagination-item.tsx
Comment (illionillion) [inline packages/components/pagination/src/pagination-item.tsx:1]: Please import `FC` from `@yamada-ui/core`

**Comment by hirotomoyamada** [comment]
Please resolve conflicts.

**Comment by hirotomoyamada** [comment]
@Utkarshrajmishra 

Please resolve conflicts.

**Comment by hirotomoyamada** [comment]
@Utkarshrajmishra

Hi, Thanks for the PR!

A week has passed since this PR was created, so maintainers will be joining this PR.

**Review by illionillion — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**